sunpy/cm/__init__.py

```diff
@@ -1,3 +1,7 @@
+from sunpy.visualization.colormaps.cm import *
```

Comment:
get out of my imports autopep8

Reply:
Should be ignored since it should skip it when the isort PR is merged in.